components/card/card.css

```diff
@@ -0,0 +1,218 @@
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  width: 80%;
+  height: 50%;
+  margin: auto;
+}
+
+.card {
+  max-width: 22rem;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  border-radius: 0.5rem;
+  color: var(--font-color);
+  background-color: var(--card-bg-color);
+  border: none;
+}
+
+.card__media {
+  height: 12.5rem;
+  width: 22rem;
+  border-radius: 0.5rem 0.5rem 0 0;
+}
+
+.card__title {
+  font-size: 1.6rem;
+  font-weight: 900;
+}
+
+.card__subtitle {
+  font-size: 1rem;
+  font-weight: 600;
+}
+
+.card__primary,
+.card__secondary {
+  padding-left: 1rem;
+}
+
+.card__primary-action:hover {
+  cursor: pointer;
+  background-color: var(--card-hover-color);
+  border-radius: 0.5rem 0.5rem 0 0;
+}
+
+.card__primary-action:hover img {
+  opacity: 0.5;
+}
+
+.card__secondary {
+  font-size: 0.97rem;
+  font-weight: 400;
+  padding-right: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.card__action-buttons {
+  padding: 1rem;
+  display: flex;
+  justify-content: space-around;
+}
+
+.card__action--button {
+  background-color: transparent;
+  color: var(--font-color);
+  border: none;
+  text-transform: uppercase;
+  font-family: inherit;
+  font-size: 1rem;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+.card__action--button:hover {
+  border-radius: 0.25rem;
+  background-color: var(--body-bg-color);
+}
+
+.card__action--icon:hover {
+  background-color: var(--body-bg-color);
+}
+
+.card__action--icon {
+  background-color: transparent;
+  border: none;
+  border-radius: 50%;
+  cursor: pointer;
+  height: 2.5rem;
+  width: 2.75rem;
+}
+
+/* Basic Card with Header */
+.card__primary-action {
+  display: flex;
+  flex-direction: column;
+}
+
+.heading-top-media {
+  order: 2;
+}
+
+.heading-top-title {
+  margin-bottom: 0;
+  order: 1;
+}
+
+.heading-top-secondary {
+  padding: 1rem 1rem 0.75rem;
+  order: 3;
+}
+
+/* Basic Card with Only Buttons */
+.hide-icon {
+  visibility: hidden;
+}
+
+/* Basic Card with Only Icons */
+.hide-button {
+  visibility: hidden;
+}
```

Comment:
You can maybe write a utility class to hide content.
Also try using webp format images instead of jpeg/png to make the images lighter
Rest all looks good!

Reply:
Good suggestion, Nikhil. These are just sample images, will use lighter ones in projects. Thanks!